packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -31,14 +39,20 @@ function InputDateTimePicker(props) {
 							id={`${props.id}-date-picker`}
 							onChange={onDateChange}
 							dateFormat={props.dateFormat}
+							useUTC={props.useUTC}
+							timezone={props.timezone}
 							value={date}
+							hideTimezone
```

Comment:
I don't see anything about timezone indication in date picker, it seems to be only in time picker.